<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Screen_14</class>
 <widget class="QWidget" name="Screen_14">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>1098</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="d2_14_backgroundWidget" native="true">
     <property name="styleSheet">
      <string notr="true">#d2_14_backgroundWidget{
 	background-color: rgb(255, 255, 255);
}

#d2_14_btnSendPhone{
	border-image: url(:/desing2/buttons/send to phone buttons/send-to-phone-button-normal.png);
}

#d2_14_btnSendPhone::pressed{
	border-image: url(:/desing2/buttons/send to phone buttons/send-to-phone-button-pressed.png);
}

#d2_14_btnTakeCoins{
	border-image: url(:/desing2/buttons/take coins buttons/take-coins-button-normal.png);
}

#d2_14_btnTakeCoins::pressed{
	border-image: url(:/desing2/buttons/take coins buttons/take-coins-button-pressed.png);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="d2_14_hlTop" stretch="0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="d2_14_btnBack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#d2_14_btnBack {
	border-image: url(:/desing2/buttons/back buttons/back-button-normal.png);
	background-color: rgba(255, 255, 255);
}

#d2_14_btnBack::pressed {
    border-image: url(:/desing2/buttons/back buttons/back-button-pressed.png);
	background-color: rgba(255, 255, 255);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="d2_14_lblBack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="d2_resource/Resource.qrc">:/desing2/buttons/back buttons/back-text-gradient.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>18</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="d2_14_lblInfoText">
          <property name="font">
           <font>
            <family>Baloo Cyrillic</family>
            <pointsize>14</pointsize>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>&lt;p style=&quot;line-height:60&quot;&gt;Qr Code!&lt;p&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="indent">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="d2_14_lblQrCode">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="d2_resource/Resource.qrc">:/desing2/qr_code/QRCode.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="d2_14_lblText">
          <property name="font">
           <font>
            <family>PT Sans</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Приложите карту.
Для отмены нажмите на терминале 
красную кнопку.
</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="d2_resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
